src/simulation/dynamics/stateSpaceEffector/stateSpaceStateEffector.cpp

```diff
@@ -0,0 +1,170 @@
+/*
+ ISC License
+
+ Copyright (c) 2024, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ */
+
+#include "stateSpaceStateEffector.h"
+#include "architecture/utilities/avsEigenSupport.h"
+#include <string>
+
+StateSpaceStateEffector::StateSpaceStateEffector(int n) : sizeOfStateSpace(n)
+{
+    this->XInit = Eigen::VectorXd::Zero(sizeOfStateSpace);
+    this->M = Eigen::MatrixXd::Zero(sizeOfStateSpace, sizeOfStateSpace);
```

Comment:
Looking further down, M does get an inverse so I think it makes sense for M

Reply:
Because of the inversion, I'm changing the M matrix to be ``ones`` instead of ``zeros``. This should make the module work without any setup, even if K is ``zeros``. Nice catch!